javascript/signin.js

```diff
@@ -67,15 +30,38 @@ function eventFocusOutPassword() {
     }
 }
 // login button click
-function eventClickBtn() {
-    if(inputEmail.value === USER_EMAIL && inputPassword.value === USER_PASSWORD) {
-        window.location.href = "/folder"
-    }
-    if(inputEmail.value !== USER_EMAIL) {
-        showErrorMessage(inputBoxEmail, "아이디를 확인해주세요.");
-    }
-    if(inputPassword.value !== USER_PASSWORD) {
-        showErrorMessage(inputBoxPassword, "비밀번호를 확인해주세요.");
+async function eventClickBtn() {
+    try {
+        // POST 메소드로 로그인 가능 여부 체크
+        const signinForm = {
+            email: inputEmail.value,
+            password: inputPassword.value
+        };
+        const response = await fetch('https://bootcamp-api.codeit.kr/api/sign-in', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(signinForm),
+        })
+        if (response.ok) {
+            const result = await response.json();
+            localStorage.setItem('accessToken', result['data']['accessToken']);
```

Comment:
r: 객체의 속성 접근 시 대괄호 표기법을 사용한 이유가 궁금해요

```suggestion
            localStorage.setItem('accessToken', result['data']['accessToken']);
```